### ISSUE TITLE
D9 chart Not correct

### ISSUE THREAD

**Issue by nanoakhi** [eiffelqiu/kundli#1]
I tried code and it works perfectly with Rashi charts but D9 charts are not correct any help in code

**Comment by nitinmukesh** [comment]
Yes same problem. D9 is not correct and also Dasha is not correct
